src/main.cc

```diff
@@ -1817,6 +1887,22 @@ GoIntoBackground()
 }
 #endif /* !_SQUID_WINDOWS_ */
 
+
+static void MasterExit()
```

Comment:
If there is no particular reason to keep MasterExit() available on Windows, then I would lower `#endif /* !_SQUID_WINDOWS_ */` to cover this function as well. Doing so will be consistent with many other local static functions that are only called from `watch_child()`. It might also help prevent errors or warnings when this signal-using code gets compiled on Windows.